test/1_test_fullpath.js

```diff
@@ -1502,13 +1488,13 @@ contract('Voucher tests - UNHAPPY PATH', async (addresses) => {
         from: users.seller.address,
       });
 
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
 
       // [1000.0100] = hex"84" = 132 = CANCELORFAULT
       assert.equal(
-        web3.utils.toHex(voucherStatus.status),
+        web3.utils.toHex(voucherStatus[0]),
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)